CMakeLists.txt

```diff
@@ -8,16 +8,20 @@ link_directories(${catkin_LIBRARY_DIRS})
 set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}  -std=c++0x" )
 
 ## This plugin includes Qt widgets, so we must include Qt like so:
-find_package(Qt4 COMPONENTS QtCore QtGui REQUIRED)
-include(${QT_USE_FILE})
+find_package(Qt5 REQUIRED COMPONENTS Widgets Core Gui)
+
+set(QT_LIBRARIES Qt5::Widgets)
```

Comment:
This line is incorrect. On Indigo/QT4, the value of `QT_LIBRATIES` was:

    /usr/lib/x86_64-linux-gnu/libQtGui.so/usr/lib/x86_64-linux-gnu/libQtCore.so

Setting it to `Qt5::Widgets` makes no sense. The variable is set by `find_package`, so you shouldn't touch it at all.